wordpress-mobile/gutenberg-mobile
# test: Update Unsupported Block tests for Appium 2

## Related PRs
- https://github.com/WordPress/gutenberg/pull/55322

## What?

Relates to #6276. Update the Unsupported Block tests for Appium 2 and
WebdriverIO.

## Why?

Improve the stability and quality of the automated tests.

## How?

- Set static device pixel ratio
 WebdriverIO (the replacement for the deprecated WD driver library) does
 not have a method for retrieving a device's pixel ratio. Therefore, this
 sets an explicit value for each of our test devices.
- test: Replace `sleep` with `pause` utility

## Testing Instructions




Local tests should pass for both platforms when running the following commands:

```
TEST_RN_PLATFORM=ios npm run device-tests:local -- gutenberg-editor-unsupported-block-visual
```

```
TEST_RN_PLATFORM=android npm run device-tests:local -- gutenberg-editor-unsupported-block-visual
```

PR submission checklist:
